### PR TITLE
[WIP]Migrate etcd storage file to markdown.

### PR DESCRIPTION
**Pull request by ptabor**
Migrating this doc: 

https://docs.google.com/document/d/1O2o1IApHWmSioXG3fez4eVlUHOrXICYGNVIzaqNS0IQ/edit?resourcekey=0-e6Iywgdkol0uiVBAaV1oww#heading=h.xye8zm7sz9wr

to repository.

Please read https://github.com/etcd-io/etcd/blob/main/CONTRIBUTING.md#contribution-flow.

### CODE REVIEW

**Comment by ahrtr** [comment]
I feel like it makes more sense to add it into `https://etcd.io/docs/v3.5/learning/` or a blog post.

**Comment by jmhbnz** [comment]
> I feel like it makes more sense to add it into `https://etcd.io/docs/v3.5/learning/` or a blog post.

Reviewing the file I agree this is something that ideally we publish to the website to be more easily discovered and consumed.

**Comment by fuweid** [comment]
Maybe we can have both markdown doc in repo and a blog post. It's really help if we can keep it up to date.

File: Documentation/contributor-guide/storage.md [hunk 1]
```diff
@@ -0,0 +1,749 @@
...
+              "modifiedIndex":3,
+              "createdIndex":3
+            }
+          }
+        ]
+      }
+    }
+  }
+}
+
```
Comment (chaochn47) [inline Documentation/contributor-guide/storage.md:727]: There is a missing end code block ``` that prevents the remaining markdown syntax to be rendered properly.

**Comment by ivanvc** [comment]
Can you assign this to me? Thanks

**Comment by jmhbnz** [comment]
> Can you assign this to me? Thanks

Hey @ivanvc - As discussed with team during etcd triage meeting please feel free to have a go migrating this document to the learning resources on the etcd website. It would be good to get it merged and accessible to people interested in learning more about etcd.

/assign @ivanvc

**Comment by ivanvc** [comment]
I just opened the PR etcd-io/website#755 addressing this. After merging that PR, I think this can be closed.

**Comment by jmhbnz** [comment]
Closing as superceded by https://github.com/etcd-io/website/pull/755.

**Pull request closed without merging**